test/metabase/api/pulse_test.clj

```diff
@@ -855,11 +855,11 @@
                  (-> ((mt/user->client :rasta) :get 200 "pulse/form_input")
                      (get-in [:channels :slack :fields])))))))
 
-    (testing "When slack is not configured, `form_input` returns just the #genreal slack channel"
+    (testing "When slack is not configured, `form_input` returns no channels"
       (mt/with-temporary-setting-values [slack-token nil]
-        (is (= [{:name "channel", :type "select", :displayName "Post to", :options ["#general"], :required true}]
+        (is (empty?
                (-> ((mt/user->client :rasta) :get 200 "pulse/form_input")
-                   (get-in [:channels :slack :fields]))))))))
+                   (get-in [:channels :slack :fields :options]))))))))
```

Comment:
I'm not sure the `:options` is valid here, since `:fields` is an array. The `empty?` test is good.

Reply:
Woof, good catch. This was silently passing since `get-in` returns `nil` when it can't find the key and `(empty? nil) ; => true`. It's now more explicit, which comes at the cost of a few extra LOC but the much larger benefit of actually failing if something goes wrong